Apeer/fiji_scripts/fijipytools.py

```diff
@@ -288,7 +277,8 @@ def readCZI(imagefile,
 
         # calc scaling in case of pyramid
         # scale = float(metainfo['Output SizeX']) / float(metainfo['SizeX'])
-        scale = float(metainfo['SizeX']) / float(metainfo['Output SizeX'])
+        scale = float(metainfo['SizeX']) / metainfo['Output SizeX']
+        # was this ported from python 2? I think you only need to float one of the inputs to the division
```

Comment:
You are right. Floating the first number is enough. If this code is intended to be only used in Py3, one could even leave out this conversion entirely. See https://stackoverflow.com/questions/1535596/what-is-the-reason-for-having-in-python

I would remove the comment though.